### PR TITLE
TELCODOCS-783 -  adding links to updated docs

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-783](https://issues.redhat.com//browse/TELCODOCS-783) -  adding links to updated docs

Version(s): 4.12

Issue: https://issues.redhat.com/browse/TELCODOCS-783

Link to docs preview:

Single node
https://54577--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno.html#installing-single-node-openshift-manually

Bare Metal
https://54577--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by slovern** [comment]
@ jerpeter1 @ jinyunma 
Following the merging of related OCP PRs on capabiities I've been able to update these finally.
I removed the notes that explained how sets of cluster capabilities are predefined, `None`,`v4.12`etc and instead added link to document from the _Install_ team that explains these
can you add an /lgtm here so I can merge?

File: installing/installing_sno/install-sno-installing-sno.adoc [hunk 1]
```diff
@@ -35,6 +35,12 @@ To install {product-title} on a single node, first generate the installation ISO
 
 include::modules/install-sno-generating-the-install-iso-manually.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../post_installation_configuration/enabling-cluster-capabilities.adoc[Enabling cluster capabilities]
+* xref:../../installing/cluster-capabilities.html#explanation_of_capabilities_cluster-capabilities[Optional cluster capabilities in OpenShift Container Platform Branch Build]
+
```
Comment (jinyunma) [inline installing/installing_sno/install-sno-installing-sno.adoc:43]: Is it possible to add two doc links with more explanation as what you did in line 116 and line 117 in installing-bare-metal.adoc?

**Review by jinyunma — COMMENTED** [review]
The update looks good to me except minor suggestion as below

File: installing/installing_sno/install-sno-installing-sno.adoc (hunk 1 above)
Comment (slovern) [reply installing/installing_sno/install-sno-installing-sno.adoc:43]: @jinyunma 
Thanks for the suggestion - yes I've done that

**Comment by jinyunma** [comment]
LGTM

**Pull request merged**